packages/containers/psammead-timestamp-container/src/index.test.jsx

```diff
@@ -63,4 +64,32 @@ describe('Timestamp', () => {
       service="news"
     />,
   );
+
+  shouldMatchSnapshot(
+    'should add alternative calendar before main date and time',
+    <Timestamp
+      timestamp={defaultTimestamp}
+      dateTimeFormat="YYYY-MM-DD"
+      format="D MMMM YYYY"
+      isRelative={false}
+      script={latin}
+      locale="fa"
+      service="persian"
+      altCalendar={jalaali}
+    />,
+  );
+
+  shouldMatchSnapshot(
+    'should not add alternative calendar before main date and time',
+    <Timestamp
+      timestamp={defaultTimestamp}
+      dateTimeFormat="YYYY-MM-DD"
+      format="D MMMM YYYY"
+      isRelative
+      script={latin}
+      locale="fa"
+      service="persian"
+      altCalendar={jalaali}
+    />,
+  );
```

Comment:
Can we nest these inside an appropriate `describe` block to give the suite a bit of structure? Also please could you make the description more descriptive, e.g. "should not add alternative calendar _if timestamp is relative_"?